install/build.py

```diff
@@ -114,6 +114,10 @@ def get_compiler_setting():
         else:
             define_macros.append(('CUPY_NO_NVTX', '1'))
 
+    cub_path = os.environ.get('CUB_PATH', '')
+    if os.path.exists(cub_path):
+        include_dirs.append(cub_path)
```

Comment:
Should it be `include_dirs.append(os.path.join(cub_path, 'include'))`?

Reply:
Thanks for your comment and sorry for my very late reply.
"include", I though I forgot adding that, but actually it is not necessary since cub does not have "include" directory. Please see https://github.com/NVlabs/cub.

Thanks!